### DISCUSSION TITLE
Added: LDM notes for April 20, 2020

### DISCUSSION THREAD

**Discussion by agocke** [dotnet/csharplang#3377] [General]
## April 20, 2020

[C# Language Design Notes for April 20, 2020](https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-04-20.md)

Records: Factories

**Comment by qrli** [comment]
> A Clone method will be generated for a record with an unspeakable type and the SpecialName flag.

"unspeakable type" or "unspeakable name"?
